components/form_inputs/overflow_buttons/ReversedOverflowButton.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <v-container>
+    <v-overflow-btn
+      class="my-2"
+      :items="items"
+      label="Overflow Btn - Reversed"
+      reverse
+      rounded
+      shaped
```

Comment:
From documentation:
shaped: Must be used with either outlined or filled

Reply:
Fixec